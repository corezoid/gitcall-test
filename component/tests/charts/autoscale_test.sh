#!/usr/bin/env bash
$CZ helm template | sed -n '/autoscale\.yaml/,/Source\:/p'
