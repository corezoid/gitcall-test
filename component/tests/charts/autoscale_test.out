# Source: gitcall_release/charts/gitcall/templates/autoscale.yaml
apiVersion: autoscaling/v1
kind: HorizontalPodAutoscaler
metadata:
  name: RELEASE-NAME-gitcall-autoscale
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "gitcall"
spec:
  minReplicas: 1
  maxReplicas: 1
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: RELEASE-NAME-gitcall-deployment
  targetCPUUtilizationPercentage: 80
